raiden/raiden_service.py

```diff
@@ -544,7 +549,13 @@ def _callback_new_block(self, latest_block):
             )
             self.handle_state_change(state_change)
 
-    def _register_payment_status(self, target, identifier, payment_type, balance_proof):
+    def _register_payment_status(
+            self,
+            target: typing.Address,
```

Comment:
please use TargetAddress

Reply:
Done. Changed the keytype of `targets_to_identifiers_to_statuses` accordingly, too.